### PR TITLE
GH-103085: Fix python locale.getencoding not to emit deprecation warning

### PR DESCRIPTION
**Pull request by youknowone**
* Issue: gh-103085

### CODE REVIEW

File: Misc/NEWS.d/next/Library/2023-03-28-15-12-53.gh-issue-103085.DqNehf.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Pure python locale.getencoding() will not warn deprecation.
```
Comment (corona10) [inline Misc/NEWS.d/next/Library/2023-03-28-15-12-53.gh-issue-103085.DqNehf.rst]:
```suggestion
Pure python :func:`locale.getencoding()`will not warn deprecation.
```

**Review by corona10 — CHANGES_REQUESTED** [review]
Thank you for catching this.
Here is a nit comment.

**Comment by youknowone** [comment]
I have made the requested changes; please review again

Thank you for the review 😄

**Review by corona10 — APPROVED** [review]
lgtm

**Pull request merged**